src/main/webapp/ChapMap.js

```diff
@@ -193,6 +193,23 @@ class ChapMap {
     this.googleMap_.panTo(coords);
   }
 
+  /**
+   * Returns a DOM element of the given type with a certain id and class
+   * @param {String} type the type of DOM element to be created
+   * @param {String} elClass the classname to be given to the element
```

Comment:
Nullable parameters have a `?` before the type in JsDoc.
Ex: `@param {?String} elClass the classname to be given to the element

Reply:
I believe I have a few more functions that also have nullable parameters so I'll update my JSDocs.